docs/chapter_array_and_linkedlist/array.md

```diff
@@ -174,7 +177,14 @@ elementAddr = firtstElementAddr + elementLength * elementIndex
 === "C"
 
     ```c title="array.c"
-    
+    /* 随机返回一个数组元素 */
+    int randomAccess(int nums[], int size) {
+        //在区间[0, nums的长度)中随机抽取一个数字
```

Comment:
Add a space between `//` and the comment.

Reply:
done